modules/iam-policy-management/src/test/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementIT.java

```diff
@@ -1194,6 +1200,33 @@ public void testListPolicyAssignments() throws Exception {
       assertNotNull(assignments);
     }
 
+    @Test(dependsOnMethods = { "testCreateS2SPolicyTemplate" })
+    public void testCreatePolicyAssignmentError() throws Exception {
+      try{
+      AssignmentTargetDetails assignmentTargetDetails = new AssignmentTargetDetails.Builder()
+        .type("Enterprise")
+        .id(testTargetEnterpriseAccountId)
+        .build();
+
+      AssignmentTemplateDetails assignmentTemplateDetails = new AssignmentTemplateDetails.Builder()
+        .id(testS2STemplateId)
+        .version(testS2SBaseTemplateVersion)
+        .build();
+    
+
+      CreatePolicyTemplateAssignmentOptions createPolicyAssignmentOptions = new CreatePolicyTemplateAssignmentOptions.Builder()
+        .version("1.0")
+        .target(assignmentTargetDetails)
+        .templates(new ArrayList<AssignmentTemplateDetails>(Arrays.asList(assignmentTemplateDetails)))
+        .build();
+      Response<PolicyAssignmentV1Collection> response = service.createPolicyTemplateAssignment(createPolicyAssignmentOptions).execute();
+      } catch (BadRequestException e) {
+            assertEquals(e.getStatusCode(), 400);
+            assertEquals(e.getMessage(), "Invalid body format. Check the input parameters. instance.target.type is not one of enum values: Account");
+        } 
+        
+    }
+
```

Comment:
Please format this code block (`testCreatePolicyAssignmentError()`) with a formatter, because it contains missing spaces (like in the `try{` statement) and wrong indentation (the whole try block) and so on.